models/idea/index.js

```diff
@@ -308,6 +308,52 @@ class Idea extends Model {
     const cursor = await this.db.query(query, params);
     return await cursor.all();
   }
+
+
+  /**
+   * Read trending ideas
+   * @param {integer} offset - pagination offset
+   * @param {integer} limit - pagination limit
+   * @returns {Promise<Idea[]>} - list of found ideas
+   */
+  static async findTrending({ offset, limit }) {
+    const now = + new Date();
```

Comment:
There is also `Date.now()` function, which returns current (milliseconds) timestamp.